### ISSUE TITLE
Alignment of the columns when some rows are fixed

### ISSUE THREAD

**Issue by Erlaunis** [handsontable/handsontable#2746]
Hi, I wanted to use the option "fixedRowsTop" to fixed my first row. Before, I tried to use "autoColumnSize" but I don't know why, it didn't work. So I found a solution by adding a css property to the div : "table-layout : auto" and the width of my columns were ok. When I added the option to fixe a row, the position of the fixed rows were completely strange and when I scrolled to the end of the grid, the cells were anormally thin. 

I made a jsfiddle to prouve it : 
http://jsfiddle.net/uLL6c0o4/1/

I know that if I use "autoColumnSize" it will work. But in my case, when I add this option, it works at the fist time and if I select one cell, every cells shrink. 

On the fiddle you can also try to scroll till the end, the cells will be really short. 
And one more thing about the scroll, when you try to scroll with the mouse, it's really really slow.

Thanks

**Comment by krzysztofspilka** [comment]
@Erlaunis so, basically what you are trying to do is using a fixed row on top and autoColumnSize at the same time? You can't just change the table-layout and expect that Handsontable will work correctly afterwards. Ultimately, you can share with us the Excel file configured the same way you would see it in Handsontable.

**Comment by Erlaunis** [comment]
@krzysztofspilka Thanks for your answer. So, if I use table-layout, I can't be sure of the result. I will do it only with handsontable by the way, without change the css. 

But it doesn't work in my case and I still don't know why. I made a pastebin with the declaration of my handsontable : http://pastebin.com/yAnLUfyz . I prefer giving all the code, with all the options. 

So I declare the handsontable like this and the "data_itk_update" is made dynamically with data in my database. But it's a really really big array, with hundred of values. 

When I display this grid, I have this first : 
![before](https://cloud.githubusercontent.com/assets/13066551/9407626/6ceabb08-480b-11e5-8937-da113ba2af91.png)

So, it's what I want but when I select one cell, I get this : 
![after](https://cloud.githubusercontent.com/assets/13066551/9407635/7f07b0a2-480b-11e5-9445-5f4edf9139c4.png)

Every cells become really short and I don't understand why :/

**Comment by krzysztofspilka** [comment]
Why would you need autoColumnSize, after all? How many columns does your data set have?

**Comment by Erlaunis** [comment]
For this grid, I have 27 columns and about 150 rows. And as you can see on my screen shot, the titles can be really large. And also, the content of some cells are "readonly" and the user can't modify it. So I need to see the entire content of the cell. That's why I really need this `autoColumnSize` :/

**Comment by krzysztofspilka** [comment]
@Erlaunis as a workaround try using:

- `autoColumnSize: true`
- `autoColumnSize: {syncLimit: '40%'},`

If you could prepare a more relevant JSFiddle showing us this strange behavior that would be very helpful.

**Comment by Erlaunis** [comment]
I think I found what is wrong with this problem. I tried again and again to reproduce this problem on jsfiddle, on blank page, and I can't do what I wanted, every time, the  display was correct, no bug, no error. So, I was thinking about the others frameworks I used. And I used jquery UI to add tabs. When I remove the tabs from my page and I included the only file with this grid, it worked. So I think there is a problem between handsontable and jquery UI. Now that I discovered this, I'll try to prepare a demo with every component to showing you the behavior I'm talking about.
